src/status_im/translations/en.cljs

```diff
@@ -76,6 +76,19 @@
    :datetime-yesterday                    "yesterday"
    :datetime-today                        "today"
 
+   :date                                  {:formats {:default "%d/%m/%Y"
+                                                     :short   "%d %b"
+                                                     :long    "%A, %d %B %Y"}
+                                            :day-names        ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
```

Comment:
Do we want to have to manage this ourselves? Are they no library handling this?

Reply:
@jeluard Yeh there are other library options. I just opted for using the i18n library already included for consistency however unfortunately it doesn't seem to have a data pack with date/time locales. 

Other options:
1) Google Closure's `goog.i18n.DateTimeFormat`: 
The issue with this is that google closure loads the appropriate locale translations based on `goog.LOCALE` at compile time. See: https://github.com/google/closure-library/blob/master/closure/goog/i18n/datetimesymbols.js#L3423-L3425. However it seems it is possible to manually override it at runtime, although i don't know how reliable that is.
2) Moment.js (https://github.com/cljsjs/packages/tree/master/moment).
3) Do you know of any others?

Happy to try out any of the above

Reply:
`moments.js` might be a good alternative. It is a too big burden to have to manage this ourselves.

Reply:
ok will try it out